src/Workspaces/SharedUtilitiesAndExtensions/Compiler/Core/NamingStyles/Serialization/NamingStylePreferencesEditorConfigSerializer.cs

```diff
@@ -39,35 +39,47 @@ public static void AppendNamingStylePreferencesToEditorConfig(
             serializableNamingRules,
             language,
             entryWriter: (name, value) => builder.AppendLine($"{name} = {value}"),
-            triviaWriter: trivia => builder.AppendLine(trivia));
+            triviaWriter: trivia => builder.AppendLine(trivia),
+            priority: null);
```

Comment:
Why no priority here?

Reply:
This is used for generating physical editorconfig file from settings. I didn't want to change the existing behavior.

Reply:
I think the default priority is fine there